src/libpriv/rpmostree-postprocess.c

```diff
@@ -372,6 +372,11 @@ process_kernel_and_initramfs (int            rootfs_dfd,
           g_ptr_array_add (dracut_argv, (char*)arg);
         }
     }
+  else
+    {
+      /* Default to this for treecomposes */
+      g_ptr_array_add (dracut_argv, (char*)"--no-hostonly");
```

Comment:
This is new behaviour, right? It makes sense to me, though let's document it in `treefile.md`?

Reply:
Sure, done in ⬇️